priv/static/phoenix_live_reload.js

```diff
@@ -0,0 +1,39 @@
+var buildFreshUrl = function(url) {
+  var date = Math.round(Date.now() / 1000).toString();
+  url = url.replace(/(\&|\\?)reload=\d*/, '');
```

Comment:
So the whole purpose of this replace is to ensure we update the url so we don't fetch the same one. Otherwise we may fetch the cached url and nothing will change. So if it worked, it worked by accident. Updating `vsn=` would be the correct thing to do.